demisto_sdk/commands/common/files/file.py

```diff
@@ -413,6 +425,7 @@ def read_from_http_request(
             timeout: timeout for the request
             handler: whether a custom handler is required, if not takes the default.
             clear_cache: whether to clear cache
+            encoding: whether to apply any custom encoding
```

Comment:
sounds like a boolean. Maybe "the encoding to apply"

Reply:
done